JupyterDocs/ariaMisclosure/ariaMisclosure_tutorial.ipynb

```diff
@@ -1,10 +1,5 @@
 {
```

Comment:
<p>There is no plot in your notebook. Please check if this is an error and needs fix in code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aria-tools/ARIA-tools-docs/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>